source/grid/grid_tools.cc

```diff
@@ -1650,7 +1650,7 @@ namespace GridTools
     // Create a local index for the locally "owned" vertices
     types::global_vertex_index next_index = 0;
     unsigned int max_cellid_size = 0;
-    std::set<std_cxx11::tuple<types::subdomain_id,types::global_vertex_index> > vertices_added;
+    std::set<std::pair<types::subdomain_id,types::global_vertex_index> > vertices_added;
```

Comment:
I don't think you can use ```std::get``` without C++11 so you also need to change those if you want to replace ```tuple``` by ```pair```.